[WIP] Simplified view organization

WrathfulSpatula opened the pull request:
This addresses issue #938. Every point of design is covered, with the exception of certain dashboard elements on the "Trends" page.

@willzeng I know you pointed to Epoch.ai for the "Trends" page design, but do you have more specific input on the contents of this view, besides the new featured tasks? I was thinking, we could create cards that list the counts of taxonomy items, respectively (tasks, methods, platforms, tags, submissions). However, I need a new back end route to query the counts for _every_ single one of those. Before I go that route, is it what you would want?

cosenal commented:
It doesn't look like we have rendering from CI on metriq-app. As a temporary solution, let's share screenshots of the new view in the description of PRs, starting from this one.

willzeng commented:
Good idea. Agreed

*Will Zeng*
President
*Unitary Fund - because evolution is unitary*

Unitary Fund <https://unitary.fund/> | Linkedin
<https://www.linkedin.com/company/unitary-fund/> | X
<https://x.com/home?lang=en> | Discord
<https://discord.com/invite/unitary-fund-764231928676089909>


On Fri, Jan 03, 2025 at 11:30 AM, Alessandro Cosentino <
***@***.***> wrote:

> It doesn't look like we have rendering from CI on metriq-app. As a
> temporary solution, let's share screenshots of the new view in the
> description of PRs, starting from this one.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/unitaryfund/metriq-app/pull/939#issuecomment-2569502255>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHZDAVLVT6EKQSPODPQICL2I23K7AVCNFSM6AAAAABUDJPVCWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRZGUYDEMRVGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

WrathfulSpatula commented:
Sorry that I dropped the ball on sharing the preview, first, but this has been on the production environment since at least last week. I take that the appearance is acceptable, but folks are opening new related issues, which are being addressed one-by-one. I'm going to merge this to unblock my workflow, but I'll continue addressing issues on the board.
